features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/themes/fancy/templates/sso/filter/template.jag

```diff
@@ -37,6 +37,7 @@
 	if (checkSSO && !Boolean(isAuthenticated)) {
 
 	    var keyStorePassword = site.ssoConfiguration.keyStorePassword;
+        var acsURL = site.ssoConfiguration.acsURL;

```

Comment:
Fixed in #1581 (https://github.com/wso2/carbon-apimgt/commit/7185e8283eb81af7fa7a750bfaeafb94bcd637ea)